src/platform_impl/linux/wayland/types/cursor.rs

```diff
@@ -0,0 +1,72 @@
+use std::sync::Arc;
+
+use cursor_icon::CursorIcon;
+use rustix::fd::OwnedFd;
+use sctk::shm::slot::{Buffer, SlotPool};
+use wayland_backend::client::ObjectData;
+use wayland_client::protocol::wl_shm::Format;
```

Comment:
We use `sctk` reexports and not `wayland_client/backend` directly.

Reply:
Fixed